plugins/ui/src/js/src/elements/Tabs.tsx

```diff
@@ -148,10 +149,13 @@ export function Tabs(props: TabComponentProps): JSX.Element {
   return (
     <DHCTabs
       onSelectionChange={onSelectionChange}
+      density="compact"
       // eslint-disable-next-line react/jsx-props-no-spreading
       {...otherTabProps}
     >
-      <TabList>{tabListChildren}</TabList>
+      <TabList UNSAFE_style={{ marginBottom: '8px' }}>
```

Comment:
Remove this

Reply:
Will change this to size-100, but will need to keep this because the python default does not apply here because in the shorthand way, we are actually never creating ui.tab_list from python, we are composing a <TabList> component   on the React side with the provided info.

Reply:
Just rechecked, and as it was originally, size-100 is not working when I pass it into react in the UNSAFE_style. If you want, we can remove this, but then there will be inconsistent styling between the two ways of declaring ui.tabs.

Reply:
That's because you should be passing it into the `marginBottom` prop, not the `UNSAFE_style` prop:
```suggestion
      <TabList marginBottom="size-100">
```